app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'user_id'
```

Comment:
- [ ] The like belongs to an author through the `author_id`, not `user_id`. Kindly remove `foreign_key: 'user_id'`.

- **[OPTIONAL]** I am making this optional because it is beyond the scope of this project but it is very important for future projects. I can see that what you have in the schema is `user_id`, instead of `author_id`. You need to run a new migration and rename it to `author_id` to match the column defined above.